layouts/partials/head/meta.html

```diff
@@ -2,10 +2,11 @@
 <link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" title="{{ .Name }}" href="{{ .Permalink | safeURL }}">
 {{ end }}
 <meta name="description" content="{{ $.Param "description" }}" />
-{{ if isset .Params "keywords" }}
-<meta name="keywords" content="{{ delimit .Keywords "," }}">
```

Comment:
I think this was also an issue: `.Keywords `. So I updated this also. Can you confirm this?

Reply:
Yes, for both of them. If you checked it with no `tags|keywords`, empty `tags|keywords` and regular `tags|keywords` it should be fine :).